plugins/by-name/lspkind/default.nix

```
@@ -23,7 +23,7 @@ lib.nixvim.plugins.mkNeovimPlugin {
     cmp = {
       enable = lib.mkOption {
         type = types.bool;
-        default = true;
+        default = config.plugins.cmp.enable;
```

```suggestion
        default = config.plugins.cmp.enable;
        defaultText = lib.literalExpression "config.plugins.cmp.enable";
```
We need to add this so that it renders nicely in the docs. You can preview the docs using `serve-docs` in the Nixvim shell.